crates/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -343,6 +349,11 @@ pub(super) fn strip(s: &Series, matches: Option<&str>) -> PolarsResult<Series> {
     }
 }
 
+#[deprecated(since = "0.32.1", note = "use polars strip_chars")]
```

Comment:
Not entirely sure what should go in the `note` here, please correct me if this is wrong